src/__test__/persistentSubscription/getPersistentSubscriptionToAllInfo.test.ts

```diff
@@ -0,0 +1,179 @@
+/** @jest-environment ./src/__test__/utils/enableVersionCheck.ts */
+
+import {
+  createTestNode,
+  delay,
+  jsonTestEvents,
+  matchServerVersion,
+  optionalDescribe,
+} from "@test-utils";
+
+import {
+  END,
+  EventStoreDBClient,
+  persistentSubscriptionToAllSettingsFromDefaults,
+  Position,
+  ROUND_ROBIN,
+  START,
+  UnsupportedError,
+} from "@eventstore/db-client";
+
```

Comment:
Similar suggestion, as in: https://github.com/EventStore/EventStore-Client-NodeJS/pull/273/files#r806800346.